initial-promises/node-server/promises.cf

```diff
@@ -437,6 +437,12 @@ bundle agent garbage_collection
         file_select => days_old("30"),
         depth_search => recurse("inf");
 
+    enterprise_edition::
```

Comment:
I don't think we should limit this to enterprise_edition. I can easily imagine Rudder being used to run on "plain" community CFEngine too. I propose we use a "fileexists("$(sys.workdir)/cf3.${sys.uqhost}.runlog")" class check to determine whether this promise should run.